tests/migration/online_migration/post_migration.pm

```diff
@@ -36,6 +36,12 @@ sub run {
     diag "SUSEConnect --status-text: $out";
     assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'" unless get_var('MEDIA_UPGRADE');
 
+    # ensable multiversion for kernel-default based on bsc#1097111, for migration continuous cases only
+    if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {
+        script_run('sed -i s/#multiversion/multiversion/g /etc/zypp/zypp.conf');
```

Comment:
As my understanding, we need disable multiversion before migration and enable it after migration. Why here want to enable it after migration?

Reply:
I think we only need make sure the multiversion is enabled during migration. If we updated the zypp.conf, we need recover it after migration. If no change, please don't update it after migration also. That's means need judge whether need update zypp.conf before migration.